flixel/tweens/FlxEase.hx

```diff
@@ -20,17 +20,21 @@ package flixel.tweens;
 class FlxEase
 {
 	/** Easing constants */
-	static var PI2:Float = Math.PI / 2;
-
-	static var EL:Float = 2 * Math.PI / .45;
-	static var B1:Float = 1 / 2.75;
-	static var B2:Float = 2 / 2.75;
-	static var B3:Float = 1.5 / 2.75;
-	static var B4:Float = 2.5 / 2.75;
-	static var B5:Float = 2.25 / 2.75;
-	static var B6:Float = 2.625 / 2.75;
-	static var ELASTIC_AMPLITUDE:Float = 1;
-	static var ELASTIC_PERIOD:Float = 0.4;
+	static var PI2(default, null):Float = Math.PI / 2;
+	static var DOUBLE_PI(default, null):Float = Math.PI * 2;
+
+	static var DEFAULT_EL(default, null):Float = Math.PI * 2 / .45;
+	static var DEFAULT_B1(default, null):Float = 1 / 2.75;
+	static var DEFAULT_B2(default, null):Float = 2 / 2.75;
+	static var DEFAULT_B3(default, null):Float = 1.5 / 2.75;
+	static var DEFAULT_B4(default, null):Float = 2.5 / 2.75;
+	static var DEFAULT_B5(default, null):Float = 2.25 / 2.75;
+	static var DEFAULT_B6(default, null):Float = 2.625 / 2.75;
+	static var DEFAULT_ELASTIC_AMPLITUDE(default, null):Float = 1;
+	static var DEFAULT_ELASTIC_PERIOD(default, null):Float = 0.4;
+	static var BOUNCE_OUT:Float->Float = bounceOutCustom();
+	static var BOUNCE_IN:Float->Float = bounceInCustom();
+	static var BOUNCE_IN_OUT:Float->Float = bounceInOutCustom();
```

Comment:
> * The prefix DEFAULT_ is verbose an unneeded
> * these should be inlined (unless they use PI, since PI isn't inlined)
> * capitalized field names should only be used for inline constants
> * Why even define a var to `bounceOutCustom` why not just reference `bounceOutCustom` directly? this just seems like needless obfuscation

* I defined it to `DEFAULT_` because it kinda looks better to me, since it (or will be) a non-changeable variable, I tried to make it clear.
* Before I tried to do it (`static inline var`), and cpp gave an error, so I thought the problem was in using inlines.
* Ok, I thought… well, I can’t make up excuses. I’ll change it.
* I get it, and I can’t tell any excuse right.